app/src/main/java/org/oppia/app/topic/play/TopicPlayFragmentPresenter.kt

```diff
@@ -50,7 +53,11 @@ class TopicPlayFragmentPresenter @Inject constructor(
     topicId = checkNotNull(fragment.arguments?.getString(TOPIC_ID_ARGUMENT_KEY)) {
       "Expected topic ID to be included in arguments for TopicPlayFragment."
     }
-    this.currentExpandedChapterListIndex = currentExpandedChapterListIndex
+    storyId = checkNotNull(fragment.arguments?.getString(STORY_ID_ARGUMENT_KEY) ?: "") {
+      "Expected topic ID to be included in arguments for TopicPlayFragment."
+    }
```

Comment:
The checkNotNull will never fail. I suggest removing the elvis operator.

Reply:
Done